homeassistant/components/tplink/light.py

```diff
@@ -288,6 +291,12 @@ def _get_light_features(self):
             # Have to make another api request here in
             # order to not re-implement pyHS100 here
             max_range, min_range = self.smartbulb.valid_temperature_range
+            # pyHS100 is abandoned, but some bulb definitions aren't present
+            # use "safe" values for something that advertises color temperature
+            if min_range == 0:
+                min_range = FALLBACK_MIN_COLOR
+            if max_range == 0:
+                max_range = FALLBACK_MAX_COLOR
```

Comment:
```suggestion
            if min_range == 0:
                min_range = FALLBACK_MIN_COLOR
                max_range = FALLBACK_MAX_COLOR
```
It's enough to check only one of these.